### PR TITLE
added object type facet #9198

### PR DESCRIPTION
**Pull request by bmaddy**
If someone could close the store when this merges, I'd appreciate it (I don't have access).

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
     t.datetime "updated_at"
   end
 
-  add_index "checksum_audit_logs", ["pid", "dsid"], name: "by_pid_and_dsid", using: :btree
+  add_index "checksum_audit_logs", ["pid", "dsid"], name: "by_pid_and_dsid"
```
Comment (carolyncole) [inline db/schema.rb:36]: Are these changes needed for the PR?

**Comment by carolyncole** [comment]
does close the store mean close the ticket?

File: db/schema.rb (hunk 1 above)
Comment (bmaddy) [inline db/schema.rb:36]: Nope. I thought the general practice was to check in any `schema.rb` changes though. Is that not the case here?

**Comment by bmaddy** [comment]
Yep, just a typo. I meant "story".
